### DISCUSSION TITLE
Setting up in Windows

### DISCUSSION THREAD

**Discussion by ml9021504** [lncrawl/lightnovel-crawler#1159] [Q&A]
Hi I would like ask on how to setup the lncrawl.exe file? And how to use it I'm a little bit stuck.

**Comment by dipu-bd** [comment]
There is no need to setup. Just download the file and double click on it to start.

You can also run it using Powershell or Command Prompt. Do <kbd>Shift</kbd> + <kbd>Right Click</kbd> on the folder container lncrawl.exe and you will see an option to `Open PowerShell window here`. Type `lncrawl.exe` to start there or `lncrawl.exe -h` for command line instructions.

Install calibre to get output in PDF

**Comment by MATCoder82** [comment]
hello,
I installed pip version of lightnovel crawler but its not open in terminal and sending to me this error:

lncrawl : The term 'lncrawl' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ lncrawl
+ ~~~~~~~
    + CategoryInfo          : ObjectNotFound: (lncrawl:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

can someone Help me to fix this error?

**Reply by USLTD** [reply]
Check where `lncrawl.exe` is installed and then check out if that path is included in `PATH` variable. If not add it

**Reply by USLTD** [reply]
Normally it should be in **Python** installation directory's `Scripts` subfolder

**Reply by alzamer2** [reply]
when you installed python , you did not check the box that ask you to add python path to your system path
there is  2 option you can do
1- run lncrawl from your python script path
2-add python script path to your system path manually, you can google "how to add path to my [system]" where [system] is your operation system like for example "how to add path to my win 10"

**Reply by dipu-bd** [reply]
add the pip script directory to environment path. check the README.md for how to. alternatively:

```python -m lncrawl```
